src/promptflow/setup.py

```diff
@@ -50,6 +50,7 @@
     "filetype>=1.2.0",  # used to detect the mime type for mulitmedia input
     "jsonschema>=4.0.0,<5.0.0",  # used to validate tool
     "docutils",  # used to generate description for tools
+    "opentelemetry-proto>=1.22.0,<2.0.0",  # used to support PFS as OpenTelemetry collector
```

Comment:
we might also need to include the exporter package?

Reply:
added, as we need to define span processor